clients/ui/frontend/src/__tests__/cypress/cypress/pages/modelRegistryView/modelVersionArchive.ts

```diff
@@ -0,0 +1,128 @@
+import { TableRow } from '~/__tests__/cypress/cypress/pages/components/table';
+import { Modal } from '~/__tests__/cypress/cypress/pages/components/Modal';
+
+class ArchiveVersionTableRow extends TableRow {
+  findName() {
+    return this.find().findByTestId('model-version-name');
+  }
+
+  findDescription() {
+    return this.find().findByTestId('model-version-description');
+  }
+
+  findLabelPopoverText() {
+    return this.find().findByTestId('popover-label-text');
+  }
+
+  findLabelModalText() {
+    return this.find().findByTestId('modal-label-text');
+  }
+
+  shouldContainsPopoverLabels(labels: string[]) {
+    cy.findByTestId('popover-label-group').within(() => labels.map((label) => cy.contains(label)));
+    return this;
+  }
+}
+
+class RestoreVersionModal extends Modal {
+  constructor() {
+    super('Restore version?');
+  }
+
+  findRestoreButton() {
+    return cy.findByTestId('modal-submit-button');
+  }
+}
+
+class ArchiveVersionModal extends Modal {
+  constructor() {
+    super('Archive version?');
+  }
+
+  findArchiveButton() {
+    return cy.findByTestId('modal-submit-button');
+  }
+
+  findModalTextInput() {
+    return cy.findByTestId('confirm-archive-input');
+  }
+}
+
+class ModelVersionArchive {
+  private wait() {
+    cy.findByTestId('app-page-title').should('exist');
+    //cy.testA11y();
```

Comment:
Need to figure out why this was failing for all of the new pages

Reply:
![image](https://github.com/user-attachments/assets/91b6302f-8370-43b1-9f62-04204fa37cda)
See the grayed out text when we are missing properties like a URI. The color contrast ratio is too low to pass our a11y standards.